.openshift-ci/build/install-gcloud-sdk.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+dnf update -y
+dnf install epel-release -y
+
+tee -a /etc/yum.repos.d/google-cloud-sdk.repo << EOM
```

Comment:
Giles have correctly mentioned, that we have quite a few different approaches implemented to install gcloud sdk. Just a note to reconcile them in the future.